x/auth/keeper/grpc_query_test.go

```diff
@@ -192,83 +194,114 @@ func (suite *KeeperTestSuite) TestGRPCQueryParameters() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestGRPCQueryModuleAccounts() {
-	var (
-		req *types.QueryModuleAccountsRequest
-	)
+func (suite *KeeperTestSuite) TestBech32Prefix() {
+	suite.SetupTest() // reset
+	req := &types.Bech32PrefixRequest{}
+	res, err := suite.queryClient.Bech32Prefix(context.Background(), req)
+	suite.Require().NoError(err)
+	suite.Require().NotNil(res)
+	suite.Require().Equal(sdk.Bech32MainPrefix, res.Bech32Prefix)
+}
 
+
+func (suite *KeeperTestSuite) TestAddressBytesToString() {
+	const addrStr = "cosmos13c3d4wq2t22dl0dstraf8jc3f902e3fsy9n3wv"
+	addrBytes := []byte{0x8e, 0x22, 0xda, 0xb8, 0xa, 0x5a, 0x94, 0xdf, 0xbd, 0xb0, 0x58, 0xfa, 0x93, 0xcb, 0x11, 0x49, 0x5e, 0xac, 0xc5, 0x30}
```

Comment:
These constants are used twice, let's put them outside of all tests

Reply:
Sure!